projects/core/bindaas-trusted-application-manager/src/main/java/edu/emory/cci/bindaas/trusted_app/impl/TrustedApplicationManagerImpl.java

```diff
@@ -12,7 +12,7 @@
 import javax.ws.rs.Path;
 import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.Response;
-import javax.xml.bind.DatatypeConverter;
+import javax.xml.bind.*;
```

Comment:
Don't unnecessarily load the whole bunch of classes when you can just load one! i.e., this change was unnecessary (unless you can explain).

Reply:
This is my bad.